res/values/strings.xml

```diff
@@ -347,13 +360,23 @@
     <!-- mailing lists -->
     <string name="mailing_list">Mailing list</string>
     <string name="ask_show_mailing_list">Show mailing list \"%1$s\"?</string>
-
+    <string name="mailing_list_profile_info">These settings (chat name and image) apply to this device only.</string>
 
     <!-- map -->
     <string name="filter_map_on_time">Show locations in time frame</string>
     <string name="show_location_traces">Show traces</string>
     <string name="add_poi">Send point of interest</string>
 
+    <!-- chat audit log -->
+    <string name="chat_audit_log_empty_message">System messages of this chat would appear here</string>
+    <string name="chat_audit_log_title">Audit Log for %1$s</string>
+    <string name="chat_audit_log_description">This view shows only System- and Info-Messages. Useful for finding the last chat actions without scrolling through many normal messages.</string>
+
+    <!-- punycode warning / labeled links -->
+    <string name="open_external_url_trust_domain">Don't ask again for</string>
```

Comment:
is a placeholder missing here? at the end of the string?

otherwise, that would need some explanations and a translator hint.